src/View/Helper/FormCollection.php

```diff
@@ -118,8 +128,10 @@ public function render(ElementInterface $element): string
         // Every collection is wrapped by a fieldset if needed
         if ($this->shouldWrap) {
             $attributes = $element->getAttributes();
-            unset($attributes['name']);
-            $attributesString = $attributes ? ' ' . $this->createAttributesString($attributes) : '';
+            if ($this->getDoctype() === Doctype::XHTML1_STRICT) {
```

Comment:
In HTML4 there is also no attribute `name` for the `fieldset` element.

Reply:
Indeed, I looked up HTML4 strict specs and you're right.
Are you able to tell me which of the supported Doctypes don't have it?

https://github.com/laminas/laminas-view/blob/c34b15f42cac410f35010163de169ab71c39bf73/src/Helper/Doctype.php#L22-L35

```php
class Doctype extends AbstractHelper
{
    /**#@+
     * DocType constants
     */
    const XHTML11             = 'XHTML11';
    const XHTML1_STRICT       = 'XHTML1_STRICT';
    const XHTML1_TRANSITIONAL = 'XHTML1_TRANSITIONAL';
    const XHTML1_FRAMESET     = 'XHTML1_FRAMESET';
    const XHTML1_RDFA         = 'XHTML1_RDFA';
    const XHTML1_RDFA11       = 'XHTML1_RDFA11';
    const XHTML_BASIC1        = 'XHTML_BASIC1';
    const XHTML5              = 'XHTML5';
    const HTML4_STRICT        = 'HTML4_STRICT';
    const HTML4_LOOSE         = 'HTML4_LOOSE';
    const HTML4_FRAMESET      = 'HTML4_FRAMESET';
    const HTML5               = 'HTML5';
    const CUSTOM_XHTML        = 'CUSTOM_XHTML';
    const CUSTOM              = 'CUSTOM';
}
```

Reply:
The `name` attribute is only allowed for HTML5 and XHTML5. The attribute was introduced with HTML5.